trio/_core/tests/test_ki.py

```diff
@@ -537,9 +537,11 @@ def test_ki_wakes_us_up():
     #   https://bugs.python.org/issue31119
     #   https://bitbucket.org/pypy/pypy/issues/2623
     import platform
-    buggy_wakeup_fd = (
-        platform.python_implementation() == "CPython" and sys.version_info <
-        (3, 6, 2)
+
+    buggy_wakeup_fd = platform.python_implementation() == "CPython" and sys.version_info < (
```

Comment:
ugh, this is an ugly one. Doesn't really matter though. It might do a better job if you put the clauses in the opposite order, and surround with parens?